inc/head.h

```diff
@@ -26,6 +27,8 @@ struct HeadState {
 
 struct Head {
 
+	struct Displ *displ;
```

Comment:
Once we are happy we can probably refactor this away. I'm not a fan of back references ;)

Reply:
Removed this in d032ac6fe6243652da073c5f4c1de3316b9d6788 in favor of a `scaling_base` integer in `Head`.